snmp/datadog_checks/snmp/data/default_profiles/cyberpower-pdu.yaml

```diff
@@ -352,7 +352,11 @@ metrics:
   - MIB: CPS-MIB
     symbol:
       OID: 1.3.6.1.4.1.3808.1.1.4.2.1.0
-      name: cyberpower.envir_temperature
+      name: cyberpower.envirTemperature
+  - MIB: CPS-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.3808.1.1.4.2.6.0
+      name: cyberpower.envirTemperatureCelsius
```

Comment:
```suggestion
```

This seems to be an addition, should we keep it for 7.49?

Reply:
@dplepage-dd included it in his fix PR so I kept it as well
it is not really a bug fix so can be added to 7.49 instead
but it makes sense to ship this with the profile as well
I don't have a strong opinion on this

Reply:
I see, let's keep it then.